pkg/kubelet/patch

```diff
@@ -0,0 +1,102 @@
+diff --git a/pkg/kubelet/kubelet.go b/pkg/kubelet/kubelet.go
```

Comment:
drop this?

Reply:
Thanks, dunno how I git-added that. Removed.